scripts/ci/cmake-v2/ci-macos11-xcode13_0-mpi.cmake

```diff
@@ -1,23 +0,0 @@
-# Client maintainer: chuck.atkins@kitware.com
```

Comment:
Was this one supposed to be deleted? It seems like there's mpi and serial versions of the other scripts. And so should there also be a `ci-macos12-xcode13_4_1-mpi.cmake`?

Reply:
Heh, nevermind. It would help if I read your earlier comment first

Reply:
Thanks @caitlinross can you approve this PR?